.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,26 @@
+name: Upload xesmf to PyPI
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.x"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools setuptools-scm wheel twine
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: |
+          python setup.py sdist bdist_wheel
+          twine upload dist/*
```

Comment:
I added this GitHub action that will automatically publish the package on PyPI. @rabernat & @JiaweiZhuang, I am happy to help out with maintaining xesmf including PyPI and conda-forge releases. Feel free to add me on PyPI: https://pypi.org/manage/project/xesmf/collaboration/ as a collaborator. My username is andersy005

Reply:
I currently don't have write permissions on this repo. So, someone with the right permissions both on PyPI and GitHub will need to set up the PYPI_USERNAME and PYPI_PASSWORD secrets.